src/tokenizer/state.js

```diff
@@ -6,20 +6,15 @@ import { types as tt } from "./types";
 
 export default class State {
   init(options: Object, input: string) {
-    this.strict = options.strictMode === false ? false : options.sourceType === "module";
+    this.strict = options.strictMode === false
+      ? false
+      : options.sourceType === "module";
 
     this.input = input;
 
     this.potentialArrowAt = -1;
 
-    this.inMethod =
-      this.inFunction =
-      this.inGenerator =
-      this.inAsync =
-      this.inPropertyName =
-      this.inType =
-      this.noAnonFunctionType =
-        false;
+    this.inMethod = this.inFunction = this.inGenerator = this.inAsync = this.inPropertyName = this.inType = this.noAnonFunctionType = false;
```

Comment:
One case I know of is Babel actually outputs code like this in the case of exports at the moment. See https://babeljs.io/repl/#?babili=false&evaluate=false&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&code=import%20thing%20from%20'bar'%3B%0A%0Aexport%20var%20a%2C%20b%2C%20c%2C%20d%2C%20e%2C%20f%2C%20g%2C%20h%2C%20i%2C%20j%2C%20k%2C%20l%2C%20m%2C%20n%20%2Co%2C%20p%2C%20q%2C%20r%2C%20s%2C%20t%2C%20u%2C%20v%2C%20x%2C%20y%2C%20z%3B&experimental=true&loose=false&spec=false&playground=true&stage=0

includes
```
exports.z = exports.y = exports.x = exports.v = exports.u = exports.t = exports.s = exports.r = exports.q = exports.p = exports.o = exports.n = exports.m = exports.l = exports.k = exports.j = exports.i = exports.h = exports.g = exports.f = exports.e = exports.d = exports.c = exports.b = exports.a = undefined;
```